views/administration/school.hbs

```diff
@@ -220,7 +220,7 @@
 							<canvas id="logo-canvas" width="35" height="35" class="hidden"></canvas>
 						</div>
 
-						{{#ifConfig "FEATURE_ADMIN_TOGGLE_STUDENT_VISIBILITY" "enabled"}}
+						{{#ifConfig "ADMIN_TOGGLE_STUDENT_VISIBILITY" "enabled"}}
```

Comment:
Also this should be changed to the new name, and should look for `true` instead of `enabled`